project-2-e-commerce/src/component/form-group/index.js

```diff
@@ -0,0 +1,19 @@
+import './form-group.scss'
+
+function FromGroup({ type, placeholder }) {
+    const renderFormGroup = (type, placeholder) => {
+        switch (type) {
+            case 'text-area':
+                return <textarea className='form-input w-100 pl-1 pt-1 fs-default' placeholder={placeholder}/>
+            default:
+                return <input type={type} className='form-input w-100 pl-1 fs-default' placeholder={placeholder}/>
+        }
+    }
+    return ( 
+        <div className='form-group mb-1'>
+            {renderFormGroup(type, placeholder)}
```

Comment:
```suggestion
            {renderFormGroup()}
```
Hàm này k cần truyền vào 2 tham số type và placeholder đâu. Nó tự lấy ở props rồi